src/MovieDetail.js

```diff
@@ -11,14 +11,14 @@ const MovieDetail = ({ details , closeMovieDetails }) => {
         <img className="backdrop" src={details.backdrop_path}/>
       <div className="copy-wrapper">
         <div className="copy">
-          <p>Tagline</p>
-          <p>Overview</p>
-          <p>Genres</p>
-          <p>Runtime</p>
+          <p>"{details.tagline}"</p>
+          <p>{details.overview}</p>
+          <p>Genres: {details.genres.join(', ')}</p>
+          <p>{details.runtime} min.</p>
```

Comment:
If you wanted to, you could do another layer of destructuring here.  Something like:

```
const { tagline, overview, genres, runtime } = details;

return (
<div>
  <p>{tagline}</p>
  <p>{overview}</p>
</div>
)
```

Let me know if that makes sense or if you have any additional questions!